tests/test_views.py

```diff
@@ -786,7 +786,7 @@ def test_results_per_page(self, rf):
         request = rf.get('/')
         response = views.json_event_search(request)
         data = json.loads(response.content)
-        assert len(data['feed']['entry']) == self.RESULTS_PER_PAGE
+        assert len(data['feed']['entry']) == min(self.RESULTS_PER_PAGE, 30)
```

Comment:
Should we split this into maybe two tests? One with less than 200 events created and one with more? It seems to me that this test was originally designed to create more events than could be seen on a single page, and thus show that only the amount per page was shown, rather than all the events. There are other tests that essentially show what we want, but if we're going for really explicit tests, maybe it's worth it?